content/en/core/releases/4.5.0.md

```diff
@@ -0,0 +1,100 @@
+
+---
+title: "4.5.0 release notes"
+linkTitle: "4.5.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.5.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#8513](https://github.com/medic/cht-core/issues/8513): Align "Target" components with Material design guidelines
+- [#8541](https://github.com/medic/cht-core/issues/8541): Options with long names look wrong in enketo selects
+- [#8686](https://github.com/medic/cht-core/issues/8686): Fix target UI to match Figma
+
+
+## Highlights
+### Schedules can be created by passing array on start_from
+We can now configure start_from to accept array of fields while configuring schedules. This allows users to pass multiple fields and first existing field will be used to create schedules. Read more about schedules in [documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/schedules/#app_settingsjson-schedules). cht-conf 3.20.0 or higher is required to validate config containing arrays of field in start_from. 
+
+### Target components are updated to match material design
+
+We updated target components to match to [material design](https://m3.material.io/components/dialogs/guidelines). Many other components were updated in [4.4.0](https://docs.communityhealthtoolkit.org/core/releases/4.4.0/#uiux-changes).
```

Comment:
Try to avoid "we" and "you" in documentation.

```suggestion
The targets UI has been updated to match [material design](https://m3.material.io/components/dialogs/guidelines). This is the next step in transitioning to using a modern and intuitive design throughout the UI. Many other components were updated in [4.4.0](https://docs.communityhealthtoolkit.org/core/releases/4.4.0/#uiux-changes).
```